packages/api/src/external/hie/set-doc-query-progress.ts

```diff
@@ -102,9 +103,20 @@ export function aggregateAndSetHIEProgresses(
 
   const aggregatedDocProgress = aggregateDocProgress(externalQueryProgresses);
 
-  const updatedDocumentQueryProgress = {
+  const updatedDocumentQueryProgress: DocumentQueryProgress = {
     ...documentQueryProgress,
-    ...aggregatedDocProgress,
+    ...(aggregatedDocProgress.convert
+      ? {
+          ...documentQueryProgress.convert,
+          ...aggregatedDocProgress.convert,
+        }
+      : {}),
+    ...(aggregatedDocProgress.download
+      ? {
+          ...documentQueryProgress.download,
+          ...aggregatedDocProgress.download,
+        }
```

Comment:
We are losing the webhookSent boolean when aggregating so we need to make sure we splat the previous doc query progress to prevent us from sending multiple webhooks

Reply:
[Follow this](https://us-west-1.console.aws.amazon.com/cloudwatch/home?region=us-west-1#logsV2:log-groups/log-group/APIInfrastructureStack-APIFargateServiceTaskDefAPIServerLogGroup35E9518F-7A2oRfufKv3Z/log-events$3Fstart$3D-43200000$26filterPattern$3D$2522Doc+conversion+status+-+patient+018e1062-6625-7487-85d0-e81197b48c9d$2522) and youll see the webhook being lost and then readded